packages/preview/tongji-undergrad-thesis-typst/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tongji-undergrad-thesis-typst"
+version = "0.1.0"
+entrypoint = "main.typ"
```

Comment:
I think the structure of your package could technically work, but it does not really follow what was intended and what other packages do. You should normally have two parts for templates such as yours:

- the core of the package, that exports a function that defines the layout of the document, and maybe some other utility functions. These functions can't be modified by users of the template. In your case, it seems that this is what you have in the `style` directory. 
- the template, that is a subdirectory that will be copied when starting a new project from this template. In your case, I think all files that are not in `style` should go in another new directory, called `template` for example. The entrypoint of the template will probably import the core of the package.

If you need more explanations of how to re-structure your repository to match that, please tell me, I can give more details.

Reply:
Could you explain the first requirement? I do not know how to realize the `unmodified` feature.Thanks